docs/releasehistory.md

```diff
@@ -55,6 +100,90 @@ print(value_roundtrip)
 # 1.0 <Unit('nanometer')>
 ```
 
+<!-- ### `ForceField.create_openmm_system()` now goes through Interchange
+
+TODO. This includes removal of some stuff from openff.toolkit.typing.engines.smirnoff.parameters:
+- NonintegralMoleculeChargeException
+- NonbondedMethod
+- ParameterHandler.assign_parameters()
+- ParameterHandler.postprocess_system()
+- ParameterHandler.check_partial_bond_orders_from_molecules_duplicates()
+- ParameterHandler.assign_partial_bond_orders_from_molecules()
+
+ -->
+
+### Breaking change: `Topology` molecule representation
+
+`Topology` objects now store complete copies of their constituent `Molecule` objects, rather than using simplified classes specific to `Topology`. This dramatically simplifies the code base and allows the use of the full `Molecule` API on molecules inside topologies.
+
+The following classes have been removed:
+- `TopologyAtom`
+- `TopologyBond`
+- `TopologyMolecule`
+- `TopologyVirtualParticle`
+- `TopologyVirtualSite`
+
+In addition, the `Topology.identical_molecule_groups` property has been added, to facilitate iterating over copies of isomorphic molecules in a `Topology`.
```

Comment:
IMO it's not necessary to create links everywhere in this document, but this one stuck out to me as an "oh I'd like to click through to see what this method does"

Reply:
Done! And good thing too, turns out the docstring needed some help to render correctly. Thanks!